Tema6/src/main/java/ListPerson.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ListPerson {
+
+    List<Person> personList = new ArrayList<Person>();
+
+
+    public void addInPerson(Person newPerson) {
+
+
+        personList.add(newPerson);
+
+
+    }
+
+    public void seeListPerson() {
+        for (Person newPerson : personList) {
+            if (newPerson != null) {
+                System.out.println(newPerson);
+            }
+
+        }
+
+    }
+
+
+    @Override
+    public String toString() {
+        return "personList: " + personList;
+    }
+
+
+    public void findElementInPerson(String startwith, String contains) {
+       System.out.println(personList.stream().map(Person::getName).filter(x -> x.startsWith(startwith)).filter(x -> x.contains(contains)).collect(Collectors.toSet()));
```

Comment:
format...list / array followed by .stream() on the same line, then a new line for each operation

Reply:
also, can the filter be done in a single line ? &&